crates/sui-core/src/execution_cache/writeback_cache.rs

```diff
@@ -877,6 +904,13 @@ impl ExecutionCacheCommit for WritebackCache {
     ) -> BoxFuture<'_, SuiResult> {
         WritebackCache::commit_transaction_outputs(self, epoch, *digest).boxed()
     }
+
+    fn commit_transactions<'a>(
```

Comment:
Also, please rename this function. I got confused by the word "commit" again. Something like "persist" would be better?

Reply:
Renamed - doc comment is in the trait defn